src/components/Layout/AuthorizationLayout/AuthorizationLayout.module.scss

```diff
@@ -0,0 +1,51 @@
+@import 'src/styles/settings.scss';
+
+.l-layout {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  height: $height-screen;
+  background-image: $background-login;
+  background-size: cover;
+  background-position: center;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    min-width: $width-small;
+    align-items: center;
+  }
+
+  &__signup {
+    margin-top: $space-m;
+  }
+
+  &__link {
+    font-weight: $fw-medium;
+    color: $color-primary;
+    text-decoration: underline;
+  }
+
+  &__logo {
+    margin-bottom: $space-l;
+    min-width: $width-medium;
+    width: $space-l;
+  }
+
+  &__title {
+    margin-bottom: $space-l;
+
+    font-size: $fw-large;
+    color: $color-white;
+  }
+
+  &__card {
+    min-width: $width-large;
+  }
+
+  &__card-item {
+    margin: $space-s 0;
+  }
```

Comment:
Card and card item should have it's own styles please remove these styles from layout